contents/site/meta/wiki_sunset.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "diaspora* Wiki Sunset"
+subtitle: The information you requested got relocated.
+---
+
+<%=
+  partial "components/inline_image",
+  alt: "We are vewwy sowwy.",
+  src: static_url("site/mr_abdoun_the_scottish_cat.jpg")
+%>
+
+In Spring 2019, we decided to discontinue the diaspora\* Project Wiki and instead merge the information with the project site and the dedicated installation guides. This allows us easier control and maintenance of the project's infrastructure, as well as streamlining contributions to both the wiki and our project sites. While we attempted to keep the old URLs working by setting up redirects, we could not redirect all wiki links. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.
+
+To help you find what you are looking for, here is a list of resources available to you:
+
+* If you need help using diaspora\*, maintaining a pod or have development related questions, check out our [guides][guides].
```

Comment:
Will this include the replacement for the tutorials? How about 'user guides' to distinguish it from 'installation guides' below?

I'm not 100% happy with 'user guides', so please discuss it amongst yourselves.

Reply:
We are fine with user guides for now, thanks. :)